Marble Blast Platinum/platinum/client/ui/AdvancedOptionsDlg.gui

```diff
@@ -113,22 +113,22 @@ function AdvancedOptionsDlg::onWake(%this) {
 $i = -1;
 $AdvancedOptions::Name    [$i++] = "alwaysShowSpeedometer";
 $AdvancedOptions::Title   [$i  ] = "Always Show Speedometer";
-$AdvancedOptions::Tooltip [$i  ] = "If enabled, the speedometer will always be visible outside of Laps levels.";
+$AdvancedOptions::Tooltip [$i  ] = "If enabled, the speedometer will no longer be restricted to Consistency and Haste levels.";
 $AdvancedOptions::Variable[$i  ] = "$pref::alwaysShowSpeedometer";
 $AdvancedOptions::Type    [$i  ] = "boolean";
 $AdvancedOptions::Name    [$i++] = "oobInsults";
 $AdvancedOptions::Title   [$i  ] = "Out of Bounds Insults";
-$AdvancedOptions::Tooltip [$i  ] = "If enabled, insults will be shown when the player goes out of bounds.";
+$AdvancedOptions::Tooltip [$i  ] = "If enabled, the player will receive special messages after going Out of Bounds a certain number of times.";
 $AdvancedOptions::Variable[$i  ] = "$pref::ShowOOBMessages";
 $AdvancedOptions::Type    [$i  ] = "boolean";
 $AdvancedOptions::Name    [$i++] = "powerupTimers";
 $AdvancedOptions::Title   [$i  ] = "PowerUp Timers";
-$AdvancedOptions::Tooltip [$i  ] = "If enabled, the remaining duration of powerups will be shown on the HUD.";
+$AdvancedOptions::Tooltip [$i  ] = "If enabled, the remaining duration of PowerUps will be shown on the HUD.";
```

Comment:
My only real nitpick here - I know the title has this with the case "PowerUp", but when we're not using title case should we still capitalize it that way? Not sure how often we do in the game. Otherwise LGTM